docs/src/description/organization.md

```diff
@@ -0,0 +1,77 @@
+# Code organization
+
+AirScript project can consist of one or more modules, each module located in a separate file with a `.air` extension. For projects consisting of multiple modules, one module must be declared as the root module, and all other modules must be declared as supporting modules.
+
+Currently, all modules must be located in a single directory, but in the future this limitation will be removed.
+
+All modules must start with a module name declaration followed by a set of source sections. These sections describe both the metadata and constraint evaluation logic for the AIR. Depending on the module type, some source sections may be required, be optional, a may not be allowed. The table below summarizes this information.
+
+| Section                                                                               | Root module | Supporting module |
+| ------------------------------------------------------------------------------------- | :---------: | :---------------: |
+| [constants](./declarations.md#constants-const)                                        | optional    | optional          |
+| [tace columns](./declarations.md#execution-trace-trace_columns)                       | required    | not allowed       |
+| [public inputs](./declarations.md#public-inputs-public_inputs)                        | required    | not allowed       |
+| [periodic columns](./declarations.md#periodic-columns-periodic_columns)               | optional    | optional          |
+| [random values](./declarations.md#random-values-random_values)                        | optional    | not allowed       |
+| [boundary constraints](./constraints.md#boundary-constraints-boundary_constraints)    | required    | not allowed       |
+| [integrity constraints](./constraints.md#integrity-constraints-integrity_constraints) | required    | not allowed       |
+| [evaluators](./evaluators.md)                                                         | optional    | required          |
```

Comment:
Evaluators aren't required in library modules, just optional, for example, you could have a module that just defines constants used across all modules